.changeset/neat-pears-fly.md

```diff
@@ -0,0 +1,5 @@
+---
+"chainlink": minor
+---
+
+#added Mode Sepolia config
```

Comment:
Could you update the changeset message here to mention that configs for mainnet and also Blast are being added?

Reply:
yup just updated